docs/pages/overview.md

```diff
@@ -3,12 +3,14 @@ layout: page
 title: Overview
 permalink: overview
 ---
-# Introducing Meshery 
-The multi-service mesh management plane adopting, operating and developing on different service meshes. 
-Meshery facilitates learning about functionality and performance of service meshes and incorporates the collection and display of metrics from applications running on or across service meshes. 
-Meshery provides this high-level functionality: 
-
-1. Performance Benchmarking
+# Introducing Meshery
+The service mesh management plane adopting, operating and developing on different service meshes. 
```

Comment:
~Adopting~ Adapting would suit better here.

Reply:
Good comment given the pervasiveness of adapters in this project. In this case, we're trying to describe part of the initial value that people get when using the project (certainly adapters to different meshes is one of those), the first of those starting with adoption of a service mesh (facilitating understanding of how they work and selection of which one to use).